sawmill-core/src/main/java/io/logz/sawmill/conditions/IpCompareCondition.java

```diff
@@ -0,0 +1,114 @@
+package io.logz.sawmill.conditions;
+
+import io.logz.sawmill.Condition;
+import io.logz.sawmill.Doc;
+import io.logz.sawmill.annotations.ConditionProvider;
+import io.logz.sawmill.parser.ConditionParser;
+import io.logz.sawmill.utilities.JsonUtils;
+
```

Comment:
Sorry accidentally resolved these 2

I remove the one unneeded linebreak and updated the packages so I am only importing what I need

import java.net.InetAddress;
import java.net.UnknownHostException;
import java.util.Map;